metrics/src/metrics.rs

```diff
@@ -208,6 +209,7 @@ impl MetricsAgent {
                 .add_field_i64("points_written", points_written as i64)
                 .add_field_i64("num_points", num_points as i64)
                 .add_field_i64("points_lost", (num_points - points_written) as i64)
+                .add_field_i64("pending_command_queue_size", cmd_queue_size as i64)
```

Comment:
I would rather name this more consistently with other fields here; such as `points_buffered`.

`command_queue` is an implementation detail, and wouldn't necessarily make sense to someone inspecting this metric on the dashboards or metrics UI.

Reply:
updated.